gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -192,6 +192,9 @@ function GM:Initialize()
 
 	hook.Run("TTT2FinishedLoading")
 
+	-- load addon language files
```

Comment:
This comment ist wrong, if you wanna load lang files on server, you are doing something wrong.
You should just make it available for the clients

Reply:
? The language systems is run on client and server since the files have to be added with `AddCSLuaFile`.

Reply:
Read the comment in detail. Making something available / downloadable isn‘t the same as loading a file